judging/scripts/create_cron_job.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Sample script for creating the cron job itself. This would only need to be done once.